src/View/Helper/FormHelper.php

```diff
@@ -2244,12 +2249,12 @@ protected function _datetimeOptions($options)
             $options['minute']['round'] = $options['round'];
         }
         unset($options['interval'], $options['round']);
-
+        
```

Comment:
You are introducing noise whitespace. Please make sure you use "trim right" for your IDE.